pkg/cmd/create/root.go

```diff
@@ -19,11 +19,11 @@ import (
 
 var (
 	// Flags
-	recreateCluster           bool
-	buildName                 string
-	kubeVersion               string
-	extraPortsMapping         string
-	kindConfigPath            string
+	recreateCluster   bool
+	buildName         string
+	kubeVersion       string
+	extraPortsMapping string
+	kindConfigPath    string
```

Comment:
I think we just fixed these in another PR. you may want to drop this change

Reply:
We should require the repo to be up-to-date for a merge, that would auto fix that. I'll drop it though